package.json

```diff
@@ -18,6 +18,7 @@
     "start:e2e": "nx run autoscript-apps:serve",
     "start:embedded-login": "nx serve embedded-login",
     "start:central-login": "nx serve central-login",
+    "start:angular-todo": "nx run angular-todo:config && nx run-many --target=serve --projects=todo-api,angular-todo",
     "start:reactjs-todo": "nx run-many --target=serve --projects=reactjs-todo, todo-api",
```

Comment:
Unfortunately, there's a bug hidden in here. The space just before `todo-api` shouldn't be there. I have a PR open for it here: https://github.com/ForgeRock/forgerock-javascript-sdk/pull/196.

Reply:
The fix PR was merged. Sorry to do this to you, but you'll need to rebase your PR again.